game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
Line is probably over 80 columns (with tabsize 4), could you move it to above the line?